assets/scripts/segments/Segment.jsx

```diff
@@ -34,7 +45,15 @@ class Segment extends React.Component {
     localeMessages: PropTypes.object,
     infoBubbleHovered: PropTypes.bool,
     descriptionVisible: PropTypes.bool,
-    suppressMouseEnter: PropTypes.bool.isRequired
+    suppressMouseEnter: PropTypes.bool.isRequired,
+    connectDragSource: PropTypes.func,
+    isDragging: PropTypes.bool,
+    connectDragPreview: PropTypes.func,
+    connectDropTarget: PropTypes.func,
+    activeSegment: PropTypes.oneOfType([
+      PropTypes.string,
+      PropTypes.number
+    ])
```

Comment:
See this example here where I ensure that only a numerical `activeSegment` is passed in as a prop: https://github.com/streetmix/streetmix/blob/71e4be4c1681aa3cb337b5bb5a0fc613946c31fb/assets/scripts/segments/SegmentDragGuides.jsx#L114

This makes it clear that we only expect to ever handle a number for this prop in the Segment component.